src/python/gudhi/representations/vector_methods.py

```diff
@@ -9,6 +9,8 @@
 #   - 2020/12 Gard: A more flexible Betti curve class capable of computing exact curves.
 #   - 2021/11 Vincent Rouvreau: factorize _automatic_sample_range
 
+import warnings
```

Comment:
If there is no more warnings.warn, probably this can go as well?

Reply:
oops, yes!